Library/Formula/spook.rb

```diff
@@ -0,0 +1,22 @@
+# Note: pull from git tag to get submodules
+class Spook < Formula
+  desc "Replacement for Guard, based on LuaJIT, MoonScript, C and libuv"
+  homepage "https://github.com/johnae/spook"
+  url "https://github.com/johnae/spook.git",
+      :tag => "0.5.5",
+      :revision => "7d1877eff3c2dd7a7f3fb5a9ad160d38ce84ba39"
+  head "https://github.com/johnae/spook.git"
+
+  depends_on "cmake" => :build
+
+  def install
+    ENV["PREFIX"] = prefix
+    ENV["CC"] = ENV.cc
+    system "make"
```

Comment:
You should be able to pass these to `make`: `system "make", "PREFIX=#{prefix}", "CC=#{ENV.cc}"`.